applications/ShapeOptimizationApplication/custom_utilities/geometry_utilities.h

```diff
@@ -18,6 +18,7 @@
 #include <string>
 #include <algorithm>
 #include <unordered_map>
+#include <functional>
 
 #include <pybind11/pybind11.h>
```

Comment:
not related to this PR but this should not be here!!!

Reply:
removed

Reply:
Sure, then this should eventually be refactored, see my reasoning above

Reply:
I will do it in a separate PR.

Reply:
I thought that i have removed this dependency, but seems i have missed this.

Reply:
#9499

Reply:
Thanks @armingeiser